src/app/shared/services/records/record-attachment.service.ts

```diff
@@ -19,7 +19,7 @@ export class RecordAttachmentService {
   createRecordAttachment(req: RecordAttachment) {
     const data = new FormData();
     data.set('record', req.record.toString());
-    data.set('path', req.path, (req.path as File).name);
+    data.set('path', req.path as File);
```

Comment:
AFAIK, after this change, the file name wouldn't be provided, please double-check it.